java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -575,6 +574,150 @@ void testReadCSV() {
     }
   }
 
+  @Test
+  void testWriteCSVToFile() throws IOException {
+    File outputFile = File.createTempFile("testWriteCSVToFile", ".csv");
+    Schema schema = Schema.builder()
+                          .column(DType.INT32, "i")
+                          .column(DType.FLOAT64, "f")
+                          .column(DType.BOOL8, "b")
+                          .column(DType.STRING, "str")
+                          .build(); 
+    CSVWriterOptions writeOptions = CSVWriterOptions.builder()
+                                               .withColumnNames(schema.getColumnNames())
+                                               .withIncludeHeader(false)
+                                               .withFieldDelimiter((byte)'\u0001')
+                                               .withRowDelimiter("\n")
+                                               .build();
+    try (Table inputTable 
+          = new Table.TestBuilder()
+              .column(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
+              .column(0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0)
+              .column(false, true, false, true, false, true, false, true, false, true)
+              .column("All", "the", "leaves", "are", "brown", "and", "the", "sky", "is", "grey")
+              .build()) {
+      inputTable.writeCSVToFile(writeOptions, outputFile.getAbsolutePath());
+      
+      // Read back.
+      CSVOptions readOptions = CSVOptions.builder()
+                                         .includeColumn("i")
+                                         .includeColumn("f")
+                                         .includeColumn("b")
+                                         .includeColumn("str")
+                                         .hasHeader(false)
+                                         .withDelim('\u0001')
+                                         .build();
+      try (Table readTable = Table.readCSV(schema, readOptions, outputFile)) {
```

Comment:
Curious to know what your thoughts are about using a 3rd party reader to read the file and then compare the output as well

Reply:
At the moment, I'm relying on Spark providing third party verification. :]

It might be good to have that extra verification, but it comes at the cost of an additional dependency.

What are the readers for the other formats (ORC/Parquet) using?